lib/libzfs/libzfs_import.c

```diff
@@ -260,6 +260,91 @@ udev_device_is_ready(struct udev_device *dev)
 }
 
 /*
+ * Wait timeout milliseconds for a newly created device to be available
```

Comment:
This is really pedantic, I know, but it should be "timeout_ms milliseconds" (with the _ms).